diesel/src/migrations/migration_error.rs

```diff
@@ -55,6 +55,7 @@ impl From<io::Error> for MigrationError {
 }
 
 #[derive(Debug, PartialEq)]
+#[cfg_attr(feature = "dev", allow(enum_variant_names))]
```

Comment:
I wasn't sure if it was a breaking change, so I left it as is. I think "QueryError" reads nicely, but if it's only used as "RunMigrationsError::QueryError" I would remove the suffix.